### PR TITLE
BAEL-6110: Fix Spring Data JPA Exception: No Property Found for Type

### PR DESCRIPTION
**Pull request by azhwani**
Simple PR

### CODE REVIEW

File: spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import com.baeldung.nopropertyfound.model.Person;
+import com.baeldung.nopropertyfound.repository.PersonRepository;
+
+@DataJpaTest
+class PersonRepositoryTest {
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryTest.java:14]: By convention tests should end in UnitTest or IntegrationTest.

File: spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Test
+    void givenQueryMethod_whenUsingValidProperty_theCorrect() {
+
+        Person person = personRepository.findByFirstName("Azhrioun");
+
+        assertNotNull(person);
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryTest.java:24]: Call to 'assertNotNull()' from 'Assert' should be replaced with a call to a method from 'org.junit.jupiter.api.Assertions'

File: spring-boot-modules/spring-boot-data-3/src/main/java/com/baeldung/nopropertyfound/repository/PersonRepository.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.baeldung.nopropertyfound.model.Person;
+
+@Repository
+public interface PersonRepository extends JpaRepository<Person, Integer> {
+
+    // Person findByFirsttName(String firstName);
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-data-3/src/main/java/com/baeldung/nopropertyfound/repository/PersonRepository.java:11]: You can remove this comment.

File: spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import com.baeldung.nopropertyfound.repository.PersonRepository;
+
+@DataJpaTest
+class PersonRepositoryIntegrationTest {
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Test
+    void givenQueryMethod_whenUsingValidProperty_theCorrect() {
```
Comment (KevinGilmore) [inline spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryIntegrationTest.java:20]:
```suggestion
    void givenQueryMethod_whenUsingValidProperty_thenCorrect() {
```

**Pull request merged**